packages/thirdweb/src/wallets/interfaces/wallet.ts

```diff
@@ -140,6 +140,12 @@ export type Wallet<TWalletId extends WalletId = WalletId> = {
    * Can be used to execute any pre-connection actions like showing a modal, etc.
    */
   onConnectRequested?: () => Promise<void>;
+
+  /**
+   * Update stored chain object
+   * @internal
+   */
+  updateChain: (chain: Chain) => void;
```

Comment:
cannot add this to the public interface, need to find a different path

Reply:
yeah, I'm considering a different solution here

Reply:
Updated